tests/publiccloud/aistack_basic.pm

```diff
@@ -0,0 +1,89 @@
+use Mojo::Base 'publiccloud::basetest';
```

Comment:
CI still claiming tidy not ok.
Before to push your changes I suggest to run in your local repo the command `tools/tidy <changed_file>`

Reply:
Fixed